src/test/java/seedu/address/logic/commands/FindCommandTest.java

```diff
@@ -74,6 +74,14 @@ public void execute_multipleKeywords_multiplePersonsFound() {
         assertEquals(Arrays.asList(CARL, ELLE, FIONA), model.getFilteredPersonList());
     }
 
+    @Test
+    public void toStringMethod() {
+        NameContainsKeywordsPredicate predicate = new NameContainsKeywordsPredicate(Arrays.asList("keyword"));
+        FindCommand findCommand = new FindCommand(predicate);
+        String expected = FindCommand.class.getCanonicalName() + "{predicate=" + predicate + "}";
```

Comment:
I believe the predicate might also benefit from a ToString() method to print the keywords for the FindCommand. Currently, the ToString() method for the FindCommand class prints the hash code of the predicate object it contains.